bin/build.sh

```diff
@@ -1,19 +1,28 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
 # Requires docker 17.05 or higher
 
+set -e
+
 echo ""
-echo "================================="
+echo "==================================="
 echo "  Building dcrbounty docker image  "
-echo "================================="
+echo "==================================="
 echo ""
 
-docker build -t decred/dcrbounty .
+
+IMAGE_NAME=decred/dcrbounty
+
+if [ "$1" != "" ]; then
+    IMAGE_NAME=$1
+fi
```

Comment:
It doesn't matter too much, but since this is using bash, this could be made more compact with parameter expansion as follows:

```sh
IMAGE_NAME="${1:-decred/dcrbounty}"
```

Reply:
Nice tip, but I actually prefer it as it is because its more simple to understand and maintain.